spec/src/main/asciidoc/changelog.adoc

```diff
@@ -25,6 +25,7 @@ Changes marked with icon:bolt[role="red"] are breaking changes relative to previ
 
 * Changes in 2.2
 ** Reverted a problematic change from 2.1 where Gauges were required to return subclasses of `java.lang.Number` 
+** Added ProcessCpuTime as a new optional base metric.
```

Comment:
to be consistent with how we showed this for other branches...
(2.2.1) Added ProcessCpuTime as a new optional base metric.